packages/clarity-visualize/src/interaction.ts

```diff
@@ -72,25 +111,32 @@ export function pointer(event: Interaction.PointerEvent, iframe: HTMLIFrameEleme
     let type = event.event;
     let doc = iframe.contentDocument;
     let p = doc.getElementById("clarity-pointer");
-    let pointerWidth = 20;
-    let pointerHeight = 24;
+    let pointerWidth = config.pointerWidth;
+    let pointerHeight = config.pointerHeight;
 
     if (p === null) {
         p = doc.createElement("DIV");
         p.id = "clarity-pointer";
         p.style.position = "absolute";
-        p.style.zIndex = "10000000";
+        p.style.zIndex = `${config.zIndex + 2}`;
```

Comment:
why +2?

Reply:
We now have three different layers for visualizing pointers:
1) Pointer arrow 
2) Canvas trail
3) Persistent click circle

I want to ensure these three layers are relatively positioned. That's why those increments.